orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/PipelineLauncher.java

```diff
@@ -81,8 +90,9 @@ private String getString(Map<String, ?> map, String key) {
   }
 
   @Override protected boolean shouldQueue(Pipeline execution) {
-    return startTracker == null ||
-      execution.getPipelineConfigId() == null ||
+    return execution.getPipelineConfigId() != null &&
```

Comment:
This is the correct logic for determining whether something should be queued ... 

queue iff:
- it's an execution of an existing pipeline (ie. has a pipelineConfigId)
- we're supposed to be limiting concurrent execution (this seems a little weird to trust what the user supplied vs. what is configured on the actual pipeline ...)
- we have a startTracker